### PR TITLE
(feat): home page alert

### LINKED ISSUE #6622: [FEATURE] Homepage alert/notice to make users aware of important information
### Description

- We have an existing alert component, could this be used `components/Alert.js`
- Load app notifications from `config/app.json` (multiple, so loop over them)
- Display on the homepage only

**json example**
```js
{
   // ...
   "alerts": [
      {
         "type": "warning",
         "message": "... ..."
      }
   ]
}
```

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by NiazMorshed2007**
### This pr introduces the feature of adding alert options in the home page discussed in #6622 

closes #6622

### `How I did?`
1. For showing alert I'm using the `components/Alert.js` component
2. Loading the data from `config/app.json` as `alertsData` at the top of the component
  ```js
  import * as alertsData from "@config/app.json";
```
3. Mapping through the `alerts` array in the `alertsData` json and returning `Alert` component
```js
{alertsData.alerts.map((alert, index) => (
  <Alert key={index} type={alert.type} message={alert.message} />
))}
```

### How does it looks like?
Here is a screen shot of the home page for this `alerts` json object in `config/app.json`
```js
// ....
 "alerts": [
    {
      "type": "warning",
      "message": "This is a warning message"
    }
  ]
  ```
  
  ### `Preview`
  
image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6892"></a>

### CODE REVIEW

File: pages/index.js
Comment (eddiejaoude) [inline pages/index.js:23]: As this is the whole config, `alertsData` is misleading

```suggestion
import * as config from "@config/app.json";
```

File: config/app.json
Comment (eddiejaoude) [inline config/app.json:7]: This should be an empty array for now, otherwise it will display after merging

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good 👍 , just a few small comments

File: config/app.json
Comment (NiazMorshed2007) [reply config/app.json:7]: Yep, removed!

File: pages/index.js
Comment (NiazMorshed2007) [reply pages/index.js:23]: Done!

File: pages/index.js
Comment (NiazMorshed2007) [reply pages/index.js:216]: Yep, solved.
The preview now:

image

**Comment by NiazMorshed2007** [comment]
Please, don't merge it now,
I just found a warning message in my terminal for importing `config`
I'm trying to fix it. 

image

**Comment by eddiejaoude** [comment]
In other places in the app we use `import app from "@config/app.json";` for the config

**Comment by NiazMorshed2007** [comment]
> In other places in

It's all fixed now,👌
You can have a look.

**Comment by eddiejaoude** [comment]
Thank you. Please can you also resolve conflicts

**Comment by NiazMorshed2007** [comment]
Done! 👍

**Review by eddiejaoude — APPROVED** [review]
Looks great, thanks 👍

**Pull request merged**
